### PR TITLE
msg/async/rdma: fix disconnect log line

### PR DESCRIPTION
**Pull request by Adirl**
Signed-off-by: Adir Lev <adirl@mellanox.com>

### CODE REVIEW

**Comment by Adirl** [comment]
@yuyuyu101 see PR

File: src/msg/async/rdma/Infiniband.cc [hunk 1]
```diff
@@ -335,7 +335,9 @@ int Infiniband::recv_msg(CephContext *cct, int sd, IBSYNMsg& im)
     r = -errno;
     lderr(cct) << __func__ << " got error " << errno << ": "
                << cpp_strerror(errno) << dendl;
-  } else if ((size_t)r != sizeof(msg)) { // valid message length
+  } else if (r == 0){ // valid disconnect message of length 0
```
Comment (yuyuyu101) [inline src/msg/async/rdma/Infiniband.cc]: minor: } else if (r == 0) { 

space between "(" and "{"

**Comment by yuyuyu101** [comment]
@Adirl hmm, plz squash these two commits... then we can merge

**Comment by Adirl** [comment]
Done

**Comment by Adirl** [comment]
@yuyuyu101  
Ooops, now should be OK

**Pull request merged**